new-components/targets/git-clone/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
```

Comment:
nit: can we move this file into a `testdata` folder so that it's clear what it's for?

Reply:
I prefer to have it at root level so it's easy to discover and acquainted people with compose know immediately what's the deal :)

We don't copy it in the containers anyways

Reply:
but it's only used for tests right? there's going to be a component.json file soon in there too, so I think ppl will get the wrong impression that we are using docker-compose under the hood

Reply:
For testing we use `dockertest`, we're not using the `docker-compose.yaml` actually - we could use the `docker-compose.yaml` but then we'd lose some control on how the test is executed - most likely we'd have to also run the integration test in the `docker-compose` and that's always not great because of startup order etc

I added it so we can run things locally - from my exp it's useful to validate behaviour locally and manually :)

I think this is a good balance